stargaze/assetlist.json

```diff
@@ -516,6 +516,31 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/XTRUMP.png"
       }
+    },
+    {
+      "description": "WOLFSHIT",
+      "denom_units": [
+        {
+          "denom": "factory/stars167y92c2fe690l0lrqyk9tahpqvu97au0cmpayh3j9455r2f6f06s78emw4/wolfshit",
+          "exponent": 0
+        },
+        {
+          "denom": "WOLFSHIT",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/stars167y92c2fe690l0lrqyk9tahpqvu97au0cmpayh3j9455r2f6f06s78emw4/wolfshit",
+      "name": "WOLFSHIT",
+      "display": "WOLFSHIT",
+      "symbol": "WOLFSHIT",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stargaze/images/wolfshit.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stargaze/images/wolfshit.png"
```

Comment:
please use an image <250 KB. This 1.4 MB

Reply:
Uploaded smaller image